config/environments/production.rb

```diff
@@ -54,7 +54,7 @@
   # config.action_cable.allowed_request_origins = [ 'http://example.com', /http:\/\/example.*/ ]
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  # config.force_ssl = true
+  config.force_ssl = true
```

Comment:
not 100% this is going to work, because the app is behind a load-balancer which does the SSL termination, but let's give it a go :)

Reply:
I could just set it so that cookies are secure and the app can still be accessed over HTTP... but I think that would mean that if the app is accessed over HTTP then the cookies wouldn't be set, so that wouldn't be much use to us.

Reply:
The load balancer makes sure there's no HTTP traffic, it redirects it to HTTPS before it even gets to the app. I think it's just when the cookies is being sent back it needs to have the header for https-only...